lib/Slider.js

```diff
@@ -719,7 +719,9 @@ var Rheostat = function (_React$Component2) {
             handleDimensions = _state3.handleDimensions;
 
 
-        return Math.max(Math.min(proposedPosition, handlePos[idx + 1] !== undefined ? handlePos[idx + 1] - handleDimensions : SliderConstants.PERCENT_FULL), handlePos[idx - 1] !== undefined ? handlePos[idx - 1] + handleDimensions : SliderConstants.PERCENT_EMPTY);
+        return Math.max(Math.min(proposedPosition, handlePos[idx + 1] !== undefined ? handlePos[idx + 1] - handleDimensions : SliderConstants.PERCENT_FULL // 100% is the highest value
+        ), handlePos[idx - 1] !== undefined ? handlePos[idx - 1] + handleDimensions : SliderConstants.PERCENT_EMPTY // 0% is the lowest value
+        );
```

Comment:
Apparently one of the changes we picked up from the airbnb master included a comment that hadn't be built yet.